content/methods/validate/metrics.md

```diff
@@ -21,16 +21,16 @@ method:
 
 ## How to do it{#how-to-metrics}
 
-1. Define the desired outcomes for your prototype. Invite stakeholders to brainstorm. Your outcomes may be based on your [design hypothesis]({{ "/methods/decide/design-hypothesis/" | url }})).
-1. List potential ways to measure whether your prototype is achieving those  outcomes. Consider quantitative and qualitative metrics:
-    1. Quantitative metrics are numerical indicators (completion rates, time on task, and click-through rates). 
-    1. Qualitative metrics capture subjective feedback and insights (user satisfaction, ease of use).
-Determine how you will collect data for each metric. This could involve user testing, surveys, analytics tools, interviews, or a combination.
-1. Choose metrics that are most effective yet easiest to measure. Be mindful that your prototype may not be the only effort being tested. People may be overwhelmed by the amount of data they’re asked to provide. Finding metrics that can be automatically collected can help relieve this burden. 
-1. Plan for who will be responsible for data collection and how often metrics will be reviewed.
-1. Establish benchmarks for each metric. These may be based on industry standards, historical data, or your desired outcomes. Benchmarks show whether the prototype meets expectations or needs further refinement. 
+1. Define the desired outcomes for your prototype. Invite stakeholders to brainstorm. Your outcomes may be based on your [design hypothesis]({{ "/methods/decide/design-hypothesis/" | url }}).
+1. List ways to measure whether your prototype is achieving those outcomes. Consider quantitative and qualitative metrics:
+    1. Quantitative metrics are numerical indicators.
+    1. Qualitative metrics capture subjective feedback and insights.
+1. Determine how you will collect data for each metric. This could involve user testing, surveys, analytics tools, and/or interviews.
```

Comment:
@mitogi would it be possible to add one or two examples back into the description of the quant and qual above?